src/usr/local/www/system_certmanager.php

```diff
@@ -794,9 +793,9 @@ function list_csrs() {
 
 	$section->addInput(new Form_Textarea(
 		'key',
-		'*Private key data',
+		'Private key data',
 		$pconfig['key']
-	))->setHelp('Paste a private key in X.509 PEM format here.');
+	))->setHelp('Paste a private key in X.509 PEM format here. In some cases, you can skip this field, for example, if your private key is on the PKCS#11 token and you only need to import and sign its certificate.');
```

Comment:
Reword to avoid addressing the user directly and over-explaining. For example:

> This field may remain empty in certain cases, such as when the private key is stored on a PKCS#11 token.

Reply:
done